template-dapp/react/README.md

```diff
@@ -0,0 +1,162 @@
+# Getting Started with Create Blocklet
+
+This project was bootstrapped with [Create Blocklet](https://github.com/blocklet/create-blocklet).
+
+This blocklet is a dapp project, which means this is a fullstacked application. It's contained both `server` and `client` code.
+
+## File Structure
+
+- .blocklet/ - blocklet bundle result
+- public/ - static files
+  - favicon.ico - favicon
+  - favicon.svg - favicon
+  - index.html - main html file, template for react
+- screenshots/ - Screenshots
+- server/ - Server side code
+  - hooks/ - `blocklet dev` hooks
+  - libs/ - Server side libraries
+  - middlewares/ - Server side middlewares
+  - routes/ - Server side routes
+  - index.js - Server side entry point
+- src/ - Client side code (A standard react app structure)
+- tools/ - Tools
+  - publish.sh - Scripts for github CI
+- .env - Environment variables
+- .env.local - Local environment variables
+- .eslintrc.js - ESLint configuration
+- .gitignore - Git ignore file
+- .prettierrc - Prettier configuration
+- blocklet.md - Blocklet README
+- blocklet.yml - Blocklet configuration
+- LICENSE - License file
+- logo.png - Blocklet logo file
+- Makefile - Makefile
+- package.json - Npm package file
+- README.md - A guide for this blocklet
+- version - Version file
+
+## Development
+
+1. Make sure you have [@abtnode/cli](https://www.npmjs.com/package/@abtnode/cli) installed
+
+   Blocklet need abtnode as a dependency. So you need to install it first.  
+   `npm install -g @abtnode/cli`  
+   See details in [https://docs.arcblock.io/abtnode/en/introduction/abtnode-setup#use-the-binary-distribution](https://docs.arcblock.io/abtnode/en/introduction/abtnode-setup#use-the-binary-distribution)
+
+2. Init abtnode & start abtnode
+
+   Before start a abtnode, you need to init abtnode.  
+   `abtnode init`  
+   `abtnode start`  
+   See details in [https://docs.arcblock.io/abtnode/en/introduction/abtnode-setup#configure-abt-node](https://docs.arcblock.io/abtnode/en/introduction/abtnode-setup#configure-abt-node)
+
+3. Go to the project directory `cd [name]`
+4. Install dependencies: `npm install` or `yarn`
+5. Start development server: `blocklet dev`
+
+## Bundle
+
+After development a blocklet, you may need to bundle it. Use `npm run bundle` command.
+
+## Deploy
+
+- If you want to deploy this blocklet to local abtnode, you can use `npm run deploy` command.
+- If you want to deploy this blocklet to remote abtnode, you can use the command below.
+
+  ```shell
+  blocklet deploy .blocklet/bundle --endpoint {your abtnode url} --access-key {abtnode access key} --access-secret {abtnode access secret}
+  ```
+
+  > Be sure you have bundle blocklet at first.
```

Comment:
Be sure you have bundle blocklet at first => Make sure the blocklet is bundled before deployment.